profiles/telegraf.nix

```diff
@@ -0,0 +1,84 @@
+{ pkgs, lib, config, ... }: let
+  isVM = lib.any (mod: mod == "xen-blkfront" || mod == "virtio_console") config.boot.initrd.kernelModules;
+  secrets = import ../secrets.nix;
+in {
+  config = {
+    systemd.services.telegraf.path = [ pkgs.nvme-cli ];
+
+    # TODO http auth
+    #deployment.keys."telegraf-secret-env".text = secrets."telegraf-secret-env";
```

Comment:
You seem to add deployment keys to the same file. Is this the preferred way?

Reply:
We have all the deployment.keys in one place, so we can avoid importing secrets.nix in multiple places.

Reply:
dropped from the profile.